metal/devices.go

```diff
@@ -17,13 +15,13 @@ import (
 )
 
 type instances struct {
-	client  *packngo.Client
+	client  *metal.DevicesApiService
```

Comment:
There's a devices_test.go file as well that will need to be updated to use equinix-sdk-go instead of packngo

Reply:
Yes updated